packages/fast-usdc/src/utils/fees.js

```diff
@@ -69,7 +67,7 @@ export const makeFeeTools = (feeConfig, customFeeConfigs = harden({})) => {
      * Calculate the split of fees between pool and contract.
      *
      * @param {Amount<'nat'>} requested
-     * @param {AccountId} [destination]
+     * @param {AccountId} destination
```

Comment:
my breaking-change sensor is blinking.
I suppose I'll trust the type checker and turn it off :)